user-console/src/main/java/org/pentaho/mantle/client/admin/UserRolesAdminPanel.java

```diff
@@ -120,12 +142,37 @@ private Widget createUsersPanel() {
     hSpacer.setWidth( "100%" );
     labelAndButtonsPanel.add( hSpacer );
     labelAndButtonsPanel.setCellWidth( hSpacer, "100%" );
-    newUserButton = new ThemeableImageButton( addButtonStyles, null, Messages.getString( "new" ) );
+    newUserButton = new ThemeableImageButton( addButtonStyles, null,"add new user"  ) {
+      @Override
+      public void onBrowserEvent(Event event) {
+        if ( event.getTypeInt() == Event.ONMOUSEUP || KeyCodes.KEY_ENTER == (char) event.getKeyCode() ) {
+          newUserOnKeyPress();
+        }
+      }
+    };
+    Roles.getButtonRole().set( newUserButton.getElement() );
+    Roles.getButtonRole().setAriaLabelProperty( newUserButton.getElement(), Messages.getString("add" ) );
+    newUserButton.getElement().setTabIndex( 0 );
+    newUserButton.getElement().setId( "add-new-user" );
+    newUserButton.addStyleName( "focus-visible" );
```

Comment:
Also, all of this could and imo should be placed on the `ThemeableImageButton` class.

Exception is the aria label property, which could be provided by a new method on the `ThemeableImageButton` class.